src/redux/api/slice.js

```diff
@@ -108,23 +109,24 @@ export const apiSlice = createApi({
           '/api/users',
           100,
           {
-            'roles[]': 'ROLE_COURIER'
-          }
+            'roles[]': 'ROLE_COURIER',
+          },
         );
 
-        return result ? { data: sortByString(result, 'username') } : { error: "result.error" };
+        if (result.error) {
+          return { error: result.error };
+        }
+        return { data: sortByString(result.data, 'username') };
       },
     }),
     getStores: builder.query({
       async queryFn(_arg, _queryApi, _extraOptions, fetchWithBQ) {
-        const result = await fetchAllRecordsUsingFetchWithBQ(
+        return await fetchAllRecordsUsingFetchWithBQ(
           fetchWithBQ,
           '/api/stores',
           100,
         );
-
-        return result ? { data: sortByName(result) } : { error: "result.error" };
```

Comment:
We need to apply `sortByName(result)` before returning.

Reply:
done